services/state/file/file.proto

```diff
@@ -29,39 +38,77 @@ option java_package = "com.hederahashgraph.api.proto.java";
 option java_multiple_files = true;
 
 /**
- * Representation of a Hedera Token Service file in the network Merkle tree.
+ * Representation of an Hedera File Service `file`.
+ *
+ * Files offer a place to store additional data, much more than is available in
+ * other entities, for use with smart contracts, non-fungible tokens, etc...
+ * As with all network entities, a file has a unique entity number, which is
+ * given along with the network's shard and realm in the form of a
+ * shard.realm.number id.
  *
- * As with all network entities, a file has a unique entity number, which is given along
- * with the network's shard and realm in the form of a shard.realm.number id.
+ * > Review Question
+ * >> File lacks auto renewal (account and period).  We need to add that here,
+ * >> and also to create and update transactions.
+ * >
+ * >> The keys field is weird and really needs to change to adminKey and
+ * >> deleteKey to be more correct and match other things.
  */
 message File {
     /**
-     * The file's unique file identifier in the Merkle state.
+     * This file's ID within the global network state.
+     * <p>
+     * This value SHALL be unique within the network.
      */
     FileID file_id = 1;
+
     /**
-      * The file's consensus expiration time in seconds since the epoch.
-      */
+     * The file's expiration time in seconds since the epoch.
```

Comment:
```suggestion
     * The file's expiration time in consensus seconds since the epoch.
```

Reply:
For expiration I believe the value is intended to be a clock time; it just might be effective slightly later due to consensus effects (or infinitely later if we never enable expiration).